src/new/mod.rs

```diff
@@ -148,19 +148,38 @@ unsafe impl<N: New> TryNew for N {
 /// This enables an `emplace()` method for [`Box`], [`Rc`], and [`Arc`]. Users
 /// are encouraged to implement this function for their own heap-allocated smart
 /// pointers.
```

Comment:
This comment needs to be moved, I think, plus a clarification on why the trait has been split.

Reply:
Done